jsonnet_test.go

```diff
@@ -349,3 +350,47 @@ func TestSetTraceOut(t *testing.T) {
 		t.Errorf("Expected %q, but got %q", expected, actual)
 	}
 }
+
+func TestGlobalBinding(t *testing.T) {
+	vm := MakeVM()
+	vm.Bind("myVar", &ast.LiteralString{Value: "bar"})
+	fileName := "main.jsonnet"
+	snippet := `{foo: myVar}`
+	expected := "{\n   \"foo\": \"bar\"\n}\n"
```

Comment:
Vysvetlim to na tomto novom teste.

Implementoval som metodu `VM.Bind`, ktora umoznuje definovat nejaky globalny identifikator a jeho hodnotu.